src/sx/src/create.js

```diff
@@ -16,12 +16,11 @@ type MediaQueries = {|
 |};
 
 // https://developer.mozilla.org/en-US/docs/Web/CSS/@keyframes
-type KeyFrames = {|
-  +from: AllCSSPropertyTypes,
-  +to: AllCSSPropertyTypes,
+export type KeyFrames = {|
```

Comment:
This type was original intended to be used differently (and arguably wrongly). Would it make sense to move it into `src/sx/src/keyframes.js`? That would allow us to clean the types here a little bit.

Reply:
Yeah, sure, I will do so.

Reply:
✔️